docs/rule/no-index-component-invocation.md

```diff
@@ -0,0 +1,58 @@
+## no-index-component-invocation
+
+We propose to allow `index`
+files inside a directory to have the equivalent semantics. Components and Component Templates can be structured as `app/components/foo-bar/index.js` and
+`app/components/foo-bar/index.hbs`. This allows additional files related to the
+component (such as a `README.md` file) to be co-located on the filesystem.
```

Comment:
While this is possible, it is not the default for Ember projects so IMHO we shouldn't be suggesting it here. We can mention that this works, but "propose" might be the wrong word here.

Reply:
Tobias, could you provide more suitable sentence?